cmd/kubeadm/app/kubeadm.go

```diff
@@ -17,21 +17,42 @@ limitations under the License.
 package app
 
 import (
+	"fmt"
 	"os"
+	"runtime"
+	"strings"
 
-	"github.com/renstrom/dedent"
 	"github.com/spf13/pflag"
 
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/util/logs"
 )
 
-var AlphaWarningOnExit = dedent.Dedent(`
-	kubeadm: I am an alpha version, my authors welcome your feedback and bug reports
-	kubeadm: please create issue an using https://github.com/kubernetes/kubernetes/issues/new
-	kubeadm: and make sure to mention @kubernetes/sig-cluster-lifecycle. Thank you!
-`)
+// TODO(phase2) use componentconfig
+// we need some params for testing etc, let's keep these hidden for now
+func getEnvParams() map[string]string {
```

Comment:
This is probably a rebase conflict, since `GetEnvParams()` is now in `cmd/kubeadm/app/api/env.go`